src/constant/settings-page.ts

```diff
@@ -0,0 +1,5 @@
+export const TAB_NAMES = {
+  general: "general",
+  health: "health",
+  personal: "personal",
+};
```

Comment:
could you put it inside the component folder, for instance components/settings/constant.ts